fgcm/fgcmChromaticity.py

```diff
@@ -267,7 +267,7 @@ def __call__(self, fitPars, returnExpGray=False):
                   (2.0 * (expGrayColorSplitWt[ok, 0] + expGrayColorSplitWt[ok, 1]) *
                    (expGrayColorSplit[ok, 0] - expGrayColorSplit[ok, 1]) *
                    (dExpGraydC0ColorSplit[ok, self.fgcmPars.expCoatingIndex[ok], 0] -
-                    dExpGraydC0ColorSplit[ok, self.fgcmPars.expCoatingIndex[ok], 1])))
+                    dExpGraydC0ColorSplit[ok, self.fgcmPars.expCoatingIndex[ok], 1])).astype(deriv.dtype))
```

Comment:
Just checking that this doesn't need to be `astype(deriv[1: ].dtype)`?

Reply:
Nope, it's just the dtype of the full array that matters.